.github/testflinger/job-def.yaml

```diff
@@ -88,16 +20,19 @@ test_data:
       git log --name-status HEAD^..HEAD
     '
 
+    # Provision charm environment
+    ssh ubuntu@$DEVICE_IP '
+      sudo ~ubuntu/intel-device-plugins-k8s-operator/.github/testflinger/provision.sh
```

Comment:
You also move the provision from cloud init to test script; can you mention it in the MR description since it is a major change ?

Reply:
Yeah, that's what I meant by the "generic enough." Let me edit the description to be more specific.

Reply:
ok that is ok then; i did not get that oemscript cannot support cloud-init; right ?

Reply:
Yeah, exactly. I linked to this in the updated PR description.